baseplate/context/thrift.py

```diff
@@ -96,22 +96,22 @@ def _call_thrift_method(self, name, *args, **kwargs):
             try:
                 with self.server_span.make_child(trace_name) as span:
                     with self.pool.connection() as prot:
-                        prot.trans.set_header("Trace", str(span.trace_id))
-                        prot.trans.set_header("Parent", str(span.parent_id))
-                        prot.trans.set_header("Span", str(span.id))
+                        prot.trans.set_header(b"Trace", str(span.trace_id).encode())
```

Comment:
what's up with these strings?

Reply:
like why are they getting bytey? the apache thrift implementation enforces that you're doing raw bytes in the headers. since we're only in the ascii range, it should be no change in wire protocol but we needed to be more clear here for the stricter code.